### PR TITLE
implemented MUX Data analytics

### PR DESCRIPTION
**Pull request by ghost**
implemented MUX Data analytics in video_player_android package

### CODE REVIEW

File: packages/video_player/video_player/pubspec.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: Flutter plugin for displaying inline video with other Flutter
 repository: https://github.com/flutter/packages/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
 version: 2.7.1
+publish_to: none
```
Comment (mrinaljain) [inline packages/video_player/video_player/pubspec.yaml]: remove this

File: packages/video_player/video_player_android/android/build.gradle [hunk 2]
```diff
@@ -6,17 +6,23 @@ buildscript {
...
         google()
         mavenCentral()
+        maven {
+            url "https://muxinc.jfrog.io/artifactory/default-maven-release-local"
+        }
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.1'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```
Comment (mrinaljain) [inline packages/video_player/video_player_android/android/build.gradle:16]: add a comment here to  maintain history

File: packages/video_player/video_player_android/android/build.gradle [hunk 3]
```diff
@@ -6,17 +6,23 @@ buildscript {
...
     }
 }
 
 rootProject.allprojects {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://muxinc.jfrog.io/artifactory/default-maven-virtual' }
+        maven { url 'https://muxinc.jfrog.io/artifactory/default-maven-release-local' }
+        maven { url 'https://muxinc.jfrog.io/artifactory/default-maven-local' }
```
Comment (mrinaljain) [inline packages/video_player/video_player_android/android/build.gradle]: please  add  comments based on required feature

File: packages/video_player/video_player_android/android/build.gradle [hunk 4]
```diff
@@ -48,7 +54,7 @@ android {
     }
 
     dependencies {
-        def exoplayer_version = "2.18.7"
```
Comment (mrinaljain) [inline packages/video_player/video_player_android/android/build.gradle:51]: add comment  here based on required  change

File: packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java [hunk 5]
```diff
@@ -79,6 +95,9 @@ final class VideoPlayer {
     ExoPlayer exoPlayer = new ExoPlayer.Builder(context).build();
     Uri uri = Uri.parse(dataSource);
 
+    System.out.println(uri.toString()+"shreyNat");
+    System.out.println(httpHeaders.toString()+"shreyNat");
```
Comment (mrinaljain) [inline packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java]: remove  this print statement

**Review by mrinaljain — CHANGES_REQUESTED** [review]
LGTM

File: packages/video_player/video_player_android/android/build.gradle (hunk 3 above)
Comment (ghost) [reply packages/video_player/video_player_android/android/build.gradle]: changes done

**Review by mrinaljain — APPROVED** [review]
LGTM

**Pull request merged**